src/components/AddNewBook.js

```diff
@@ -30,25 +30,33 @@ const AddNewBook = ({ addBookToStoreProps }) => {
   };
 
   return (
-    <form>
-      <input
-        type="text"
-        name="title"
-        placeholder="Book title"
-        onChange={onChange}
-        value={formInputs.title}
-        required
-      />
-      <input
-        type="text"
-        name="category"
-        placeholder="Category"
-        onChange={onChange}
-        value={formInputs.category}
-        required
-      />
-      <button type="button" onClick={addBook}>Add New Book</button>
-    </form>
+    <>
+      <p className="add-form-header">ADD NEW BOOK</p>
+      <form className="add-form">
+        <div className="add-form__left">
```

Comment:
- [ ] Please make sure your project UI matches the one provided on Zeplin. Please compare the images and make the necessary changes
<img width="1268" alt="Screen Shot 2021-10-09 at 4 07 54 PM" src="https://user-images.githubusercontent.com/47994600/136665964-712cb1b0-8f0c-4698-a491-dd241a4a4f83.png">
<img width="998" alt="Screen Shot 2021-10-09 at 4 08 21 PM" src="https://user-images.githubusercontent.com/47994600/136665984-49bbb862-ee25-4e67-a45b-94908bc93dca.png">